packages/contracts/contracts/tutorials/Arbitrage.sol

```diff
@@ -0,0 +1,103 @@
+pragma solidity ^0.4.19;
+pragma experimental ABIEncoderV2;
+
+import "../Exchange.sol";
+import "../tokens/Token.sol";
+import "./EtherDelta.sol";
+import "../tokens/WETH9.sol";
+
+contract Arbitrage is Ownable {
+
+    Exchange exchange;
+    EtherDelta etherDelta;
+    address proxyAddress;
+
+    uint constant MAX_UINT = 2**256 - 1;
+
+    function Arbitrage(address _exchangeAddress, address _etherDeltaAddress, address _proxyAddress) {
+        exchange = Exchange(_exchangeAddress);
+        etherDelta = EtherDelta(_etherDeltaAddress);
+        proxyAddress = _proxyAddress;
+    }
+
+    function setAllowances(address tokenAddress) public onlyOwner {
```

Comment:
Can we add some comments on when to set an allowance? It's also not obvious that you need to deposit/withdraw from the Arbitrage contract (and how to do that). While you can technically do this manually by setting an allowance and then calling `transfer` and `transferFrom` directly on the token, it would be more intuitive to have explicit `withdraw` and `deposit` methods.

Also, let's make this `external`.